client.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
Client should not be present in this repo. This is the server repository. Open a repository for the client and move the client code there.